src/app/learn/reference/configuration/additional-config-files/index.html

```diff
@@ -83,11 +83,39 @@
 
   <p>
-    If you need to track the <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> querystring parameters and expose them through the <stache-code>SkyAppConfig</stache-code> for local development, we recommend creating a <stache-code>skyuxconfig.serve.json</stache-code> file and adding the <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> settings to the <stache-code>params</stache-code> property. In addition, if you use the global nav exclusively and don’t want the local nav alongside it, you can set the <stache-code>routes</stache-code> property to <stache-code>false</stache-code>.
+    <a routerLink="/learn/reference/configuration">The <stache-code>params</stache-code> configuration property</a> allows you to specify querystring parameters to track and exposes them through <stache-code>SkyAppConfig</stache-code>. To apply parameters to specific SKY UX CLI commands or apply specific values to CLI commands, you can create <stache-code>skyuxconfig.[COMMAND].json</stache-code> files and add your querystring parameters to the <stache-code>params</stache-code> property.
+  </p>
+  <p>
+    For example, if you use a specific environment ID and service ID for testing, you can specify default <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> values for local development in <stache-code>skyuxconfig.serve.json</stache-code>. This allows you to include the environment ID and service ID for testing in the querystring when you serve without pasting the values in the URL. In addition, if you use the global nav exclusively and don’t want the local nav alongside it, you can set the <stache-code>routes</stache-code> property to <stache-code>false</stache-code>.
+  </p>
+  <stache-code-block>
+{
+  "routes": false,
+  "params": {
+    "envid": {
+      "value": "[default envid]"
+    },
+    "svcid": {
+      "value": "[default svcid]"
+    }
+  }
+}
+  </stache-code-block>
+
+  <stache-page-anchor>
+    <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> settings
+  </stache-page-anchor>
+
+  <p>
+    To track <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> querystring parameters and expose them through <stache-code>SkyAppConfig</stache-code> for a particular CLI command, create a <stache-code>skyuxconfig.[COMMAND].json</stache-code> file and add the <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> settings to the <stache-code>params</stache-code> property.
+  </p>
+
+  <p>
+    For example, if you need to track the <stache-code>svcid</stache-code> and <stache-code>envid</stache-code> querystring parameters for local development, create a <stache-code>skyuxconfig.serve.json</stache-code> file and add the settings for local development runs. In addition, if you use the global nav exclusively and don’t want the local nav alongside it, set the <stache-code>routes</stache-code> property to <stache-code>false</stache-code>.
```

Comment:
This paragraph sounds really similar to the one a couple above it.

Reply:
I didn't mean to leave this section in place. I'd copied and pasted it to be able to compare what we had with the new Parameters section, but I think the Parameters section should suffice and that we shouldn't need a section for svcid and envid. I'll delete this section and update the PR.